### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by daniel17903**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -3,5 +3,10 @@
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "[svelte]": {
     "editor.defaultFormatter": "svelte.svelte-vscode"
+  },
+  "eslint.format.enable": true,
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true,
+    "source.organizeImports": true
```
Comment (skjerns) [inline .vscode/settings.json:10]: Ah great, this will enforce the same code styling in both of our VSCodes?

Comment (daniel17903) [reply .vscode/settings.json:10]: Yes, I think it at least synchronizes some settings for code formatting plugins and import organization automatically for everyone. However I think you can still have different settings within these plugins.

**Review by skjerns — APPROVED** [review]
No functional change as far as I see, so all good :)

**Comment by muety** [comment]
Any updates on this? 😃 Ready to be merged?

**Pull request closed without merging**